components/supersaas/sources/changed-appointments.js

```diff
@@ -0,0 +1,39 @@
+const supersaas = require('../supersaas.app.js');
+
+module.exports = {
+  key: 'supersaas-changed-appointments',
+  name: 'New or changed appointments',
+  description: `This event source tracks new and changed appointment events from the selected schedules and emits them.`,
+  version: '0.0.1',
+
```

Comment:
Can you remove the empty newlines between sections here and in the rest of the files? We need to develop ESLint / Prettier rules that automate this, so that should catch this in the future.

Reply:
Are empty lines *always* undesirable, or only inside JSON and object literals? Should the line after the `TODO` comment be kept? What about the other ones from other function bodies?

Reply:
In this case I'd only suggest removing the empty lines between properties of the Node module (e.g. between `version` and `props`, and between `props` and `hooks`. Other empty newlines within methods, for example, are totally fine.